syft/execution/plan.py

```diff
@@ -95,6 +97,8 @@ class Plan(AbstractObject):
         owner: plan owner
         tags: plan tags
         description: plan description
+        base_framework: The underlying framework (pytoch, tensorflow) which the plan is to be executed in
+        frameworks: A list of frameworks which the plan will also support
```

Comment:
Seems like `frameworks` might be left over from an intermediate form of this PR or something?